content/departments/product-engineering/engineering/cloud/devops/managed/index.md

```diff
@@ -83,6 +84,29 @@ These managed Docker Compose deployments can scale up to the largest GCP instanc
 
 We do not offer Kubernetes managed instances today as this introduces some complexity for us in terms of ongoing maintenance and overhead, we may revisit this decision in the future.
 
+### Environments
```

Comment:
@michaellzc I moved your docs regarding testing environment from operations to main MI page (agreed with Raf that it's very important so should live on first page) and added some small extension for v1.0 & v1.1 - hopefully I moved all correct.

Reply:
yep! this looks good to me